### PR TITLE
chore: deploying 2020.08 and WAF to KF PROD

### PR DESCRIPTION
**Pull request by atharvar28**
applying 2020.08 to KF PROD
fence -> 4.23.2
deploying WAF

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# data.kidsfirstdrc.org/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)
- portal
  - Portal should be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

File: data.kidsfirstdrc.org/manifest.json [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
-    "arborist": "quay.io/cdis/arborist:2020.05",
-    "fence": "quay.io/cdis/fence:2020.05",
-    "indexd": "quay.io/cdis/indexd:2020.05",
-    "peregrine": "quay.io/cdis/peregrine:2020.05",
-    "pidgin": "quay.io/cdis/pidgin:2020.05",
-    "revproxy": "quay.io/cdis/nginx:2020.05",
-    "sheepdog": "quay.io/cdis/sheepdog:2020.05",
-    "portal": "quay.io/cdis/data-portal:2020.05",
+    "arborist": "quay.io/cdis/arborist:2020.08",
+    "fence": "quay.io/cdis/fence:2.43.2",
```
Comment (paulineribeyre) [inline data.kidsfirstdrc.org/manifest.json:10]:
```suggestion
    "fence": "quay.io/cdis/fence:4.23.2",
```

Comment (atharvar28) [reply data.kidsfirstdrc.org/manifest.json]: 🤦

**Comment by PlanXCyborg** [comment]
# data.kidsfirstdrc.org/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)
- portal
  - Portal should be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Review by paulineribeyre — APPROVED** [review]
lgtm

**Comment by PlanXCyborg** [comment]
# data.kidsfirstdrc.org/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)
- portal
  - Portal should be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Pull request merged**